charts/civil-sdt/values.preview.template.yaml

```diff
@@ -13,7 +13,7 @@ java:
     enabled: true
   keyVaults:
     civil:
-      resourceGroup: civil
+      resourceGroup: civil-service
```

Comment:
is the changed to civil-service?

Reply:
Can't find any reference to resourceGroup in [key vault documentation](https://github.com/hmcts/chart-library#keyvault-secret-csi-volumes).  Found that it's not used in other repositories (e.g. [ccd-definition-store-api](https://github.com/hmcts/ccd-definition-store-api/blob/master/charts/ccd-definition-store-api/values.preview.template.yaml#L8)), so removed it.